### ISSUE TITLE
Crafty.load() hangs when supplied with an array of strings specifying assets

### ISSUE THREAD

**Issue by darkodraskovic** [craftyjs/Crafty#910]
When one supplies an array of strings specifying resources to Crafty.load(), a game simply hangs without outputing any error or in other way indicating asset loading failure. 

This is a misleading behavior, suggesting that assets are loading, while the opposite is true.

**Comment by starwed** [comment]
Yes, this is especially problematic because we changed the syntax for loading assets -- there are almost definitely old tutorials out there which use the old style, so we should either support that or explicitly note that it no longer works.  (I'd favor the latter.)

**Comment by darkodraskovic** [comment]
I think that there should be at least some error message. Ideally, this error message should explain how things have changed and direct a potential user to API Crafty.Loader page. 

Anyway, I've made a small change to the code given in this tutorial: http://buildnewgames.com/introduction-to-crafty/ and it works just fine with v. 0.6. 4. I've only had to change an asset specification string to a JSON formated object and to delete a part of the code that adds sound mappings (now integrated in JSON asset object). From reading comments, I've realized that people think you introduce code breakers all the time with every new minor version (at least, that is the opinion of the author of the tutorial) and this drives off people. However, in reality it seems to be a minor issue. Especially if you say it out loud here and/or on the official Crafty website. 

The first thing I would do, would be to ask the author of the aforementioned tutorial for the permission to rework the tutorial and republish it as the official guide with all credits due. I've read it in parallel with the API and had no problem to understand how Crafty works. Without it though, I'm not sure that I would have gotten it, despite the ok API docs.

**Comment by shepardtone** [comment]
I had the same experience as @darkodraskovic and, as a newcomer to Crafty, would have been stuck had I not seen his fix in the tutorial comments.

**Comment by starwed** [comment]
Ok, so the current situation is:
- We fixed  `Crafty.load` to warn if you use the old syntax (#911), so it doesn't just mysteriously hang.   So when we release a new version that should mitigate things
- Where the tutorial is linked to from our site, I added a note about the issue
- And I just emailed Darren about us hosting the tutorial

If for some reason that doesn't work out, I guess we'll just have to write a new one!  I've already expanded the [documentation](http://craftyjs.com/documentation/) section with more detailed descriptions of basic concepts, so hopefully that also helps complement the API docs.
